### DISCUSSION TITLE
How do I trigger a self-hosted ECS Deploy Runner via GitHub Actions?

### DISCUSSION THREAD

**Discussion by zackproser** [gruntwork-io/knowledge-base#641] [How do I]
A customer asked: 
> We have deployed an ecs deploy runner using this: https://github.com/gruntwork-io/terraform-aws-ci/tree/main/modules/ecs-deploy-runner

My question is how can we utilize this `ecs runner` to run the workflow, which I have already created in our test repo. 

---

<ins datetime="2023-01-20T15:31:22Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109814">Tracked in ticket #109814</a></p>
</ins>

**Comment by denis256** [comment]
Hi,
I think the approach is the same as in documentation - through `infrastructure-deployer` CLI application in which is passed region, function name, and commands to execute

References:

https://github.com/gruntwork-io/terraform-aws-ci/blob/main/modules/infrastructure-deployer/core-concepts.md#how-do-i-invoke-the-ecs-deploy-runner

https://github.com/gruntwork-io/terraform-aws-ci/blob/main/modules/infrastructure-deployer/core-concepts.md#how-do-i-install-the-infrastructure-deployer-cli

https://github.com/gruntwork-io/terraform-aws-ci/tree/main/modules/infrastructure-deployer
